RELEASE-NOTES.md

```diff
@@ -1,5 +1,213 @@
 # Prefect Release Notes
 
+## Release 3.0rc1
+
+We're excited to announce the release candidate of Prefect 3.0. It's the most flexible, powerful, fastest version of Prefect yet. Prefect 3.0 includes several exciting new features. Install it by running `pip install prefect==3.0rc1` and check out the docs [here](https://docs-3.prefect.io/3.0rc/getting-started/index).
+
+### Run tasks independently of flows
+You can now run, serve, and submit tasks outside of flows and inside of other tasks.
+
+```python
+from prefect import task
+
+@task
+def my_background_task(name: str):
+    print(f"Hello, {name}!")
+
+if __name__ == "__main__":
+    my_background_task.delay("ford")
+```
+
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/13695
+- https://github.com/PrefectHQ/prefect/pull/13692
+- https://github.com/PrefectHQ/prefect/pull/13681
+- https://github.com/PrefectHQ/prefect/pull/13662
+- https://github.com/PrefectHQ/prefect/pull/13653
+- https://github.com/PrefectHQ/prefect/pull/13643
+- https://github.com/PrefectHQ/prefect/pull/13589
+- https://github.com/PrefectHQ/prefect/pull/13684
+- https://github.com/PrefectHQ/prefect/pull/13676
+- https://github.com/PrefectHQ/prefect/pull/13276
+- https://github.com/PrefectHQ/prefect/pull/13611
+- https://github.com/PrefectHQ/prefect/pull/13547
+
+### Transactional semantics
+
+Rollback and commit hooks that facilitate idempotent python code
```

Comment:
it might be nice for this one to be in a sentence format like the other highlights